CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.2] - TBD
+### Added
+- Added Travis-CI pipeline to verify Unit/Integration tests on push & PR
```

Comment:
Since this is an internal feature that end users don't see or care about it's one of the rare cases that we don't update the changelog.